docs/source/composer_model.rst

```diff
@@ -100,25 +100,27 @@ To compute metrics during training, implement the following methods:
 
 .. code:: python
 
-   def validate (self, batch) -> outputs, targets:
+   def eval_forward (self, batch, outputs) -> outputs:
        ...
 
-   def metrics(self, train=False) -> Metrics:
+   def get_metrics(self, is_train=False) -> Dict[str, Metric]:
```

Comment:
missing description of `update_metrics`?

Reply:
This was marked as resolved, but I don't see additional text here? Is there a commit missing?